src/objective-c/examples/SwiftSample/ViewController.swift

```diff
@@ -59,7 +59,7 @@ class ViewController: UIViewController {
     }
 
     // TODO(jcanizales): Revert to using subscript syntax once XCode 8 is released.
-    RPC.requestHeaders.setObject("My value", forKey: "My-Header")
+    RPC.requestHeaders.setObject("My value", forKey: "My-Header" as NSCopying)
```

Comment:
Can we already use `RPC.requestHeaders["My-Header"] = "My value"`? Same for `call.requestHeaders` below.